LoopUI/ViewModel/CGMStatusHUDViewModel.swift

```diff
@@ -101,6 +97,20 @@ public class CGMStatusHUDViewModel {
     init(staleGlucoseValueHandler: @escaping () -> Void) {
         self.staleGlucoseValueHandler = staleGlucoseValueHandler
     }
+
+    private var lastCommunicationDate: Date? {
+        didSet {
+            if let signalLossHighlight = signalLossHighlight {
+                statusHighlight = signalLossHighlight
+            }
+        }
+    }
+
+    private var staleGlucoseAge: TimeInterval?
+
+    func setLastCommunicationDate(_ lastCommunicationDate: Date?) {
```

Comment:
[nit] Why not just allow access to `lastCommunicationDate` directly? There's nothing really being done in `setLastCommunicationDate`.